Automattic/newspack-plugin
# feat/campaigns analytics date range

### All Submissions:






### Changes proposed in this Pull Request:



- Closes #991
- Adds a new component - `DateRangePicker` (@thomasguillot - this needs a design review, see the imported component's CSS [here](https://github.com/onefinestay/react-daterange-picker/blob/master/src/css/react-calendar.scss))
- Fixes `Modal` styling

### How to test the changes in this Pull Request:

1. On `master`, observe `Modal` styling is a bit off - padding is too large (probably WP 5.8 related)
2. Switch to this branch, observe improved styling
3. Visit the Campaigns Wizard, Campaigns tab observe conversion rate & viewability displaying correctly
4. Visit the Analytics tab, observe a button with the date range
5. Click the button, a modal with a date range picker should appear
6. Choose a date range, observe the chart & data updated

### Other information:
